warehouse/index.js

```diff
@@ -52,6 +52,9 @@ const getDataType = val => {
   if (validTimestamp(val)) {
     return "datetime";
   }
+  if (options.getDataTypeOverride && typeof options.getDataTypeOverride === "function") {
+    return options.getDataTypeOverride(val, options);
```

Comment:
name the func string specific? this will override only the string datatype right

Reply:
yes. but kept it generic